packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -1027,6 +1028,18 @@ void runApp(Widget app) {
     ..scheduleWarmUpFrame();
 }
 
+/// Debug-only functionality used to perform faster hot reloads.
+///
+/// This field is set by expression evaluation in the flutter tool and is
```

Comment:
Maybe add a little more documentation here? E.g. it appears that this has to be set every time before  fastReassemble is invoked?

Reply:
Would also be good to document what `cb` needs to do.

Reply:
I'm not sure how best to document this. Unlike other APIs, no one should ever set this themselves (and if it is set it will get blown away)

Reply:
The documentation wouldn't be for people writing a Flutter app, but for people working on the framework or the tool when looking at this again in the future :P

Reply:
Updated, PTAL